src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$appSideMargin: 20px;
+$editFieldeftMargin: 43px;
```

Comment:
The variable name 'editFieldeftMargin' seems to be a typo. Consider renaming it to 'editFieldLeftMargin' for better readability.

Reply:
Done.